packages/features/bookings/Booker/Booker.tsx

```diff
@@ -0,0 +1,196 @@
+import { FiCalendar, FiColumns, FiGrid } from "@calcom/ui/components/icon";
+
+import { AvailableTimeSlots } from "./components/AvailableTimeSlots";
+import { Away } from "./components/Away";
+import { BookEventForm } from "./components/BookEventForm";
+import { DatePicker } from "./components/DatePicker";
+import { EventMeta } from "./components/EventMeta";
+import { LargeCalendar } from "./components/LargeCalendar";
+import { BookerSection } from "./components/Section";
+import { fadeInUp, fadeInLeft, resizeAnimationConfig } from "./config";
+import { useBookerStore, useInitializeBookerStore } from "./store";
+import type { BookerLayout, BookerProps } from "./types";
+import { useGetBrowsingMonthStart } from "./utils/dates";
+import { useEvent } from "./utils/event";
+
+// @TODO: Test embed view
+/* @TODO: eth signature / gates */
+
+const BookerComponent = ({ username, eventSlug, month, rescheduleBooking }: BookerProps) => {
+  const [browsingMonthStart, setBrowsingMonthStart] = useGetBrowsingMonthStart(month);
+  // Custom breakpoint to make calendar fit.
+  const isMobile = useMediaQuery("(max-width: 800px)");
+  const StickyOnDesktop = isMobile ? Fragment : StickyBox;
+  const rescheduleUid =
+    typeof window !== "undefined" ? new URLSearchParams(window.location.search).get("rescheduleUid") : null;
+  const event = useEvent();
+  const [layout, setLayout] = useBookerStore((state) => [state.layout, state.setLayout], shallow);
+  const [bookerState, setBookerState] = useBookerStore((state) => [state.state, state.setState], shallow);
+  const [selectedDate, setSelectedDate] = useBookerStore(
+    (state) => [state.selectedDate, state.setSelectedDate],
+    shallow
+  );
+  const [selectedTimeslot, setSelectedTimeslot] = useBookerStore(
+    (state) => [state.selectedTimeslot, state.setSelectedTimeslot],
+    shallow
+  );
+
+  useInitializeBookerStore({
+    username,
+    eventSlug,
+    month: browsingMonthStart.toDate(),
+    eventId: event?.data?.id,
+    rescheduleUid,
+    rescheduleBooking,
+  });
+
+  const onMonthChange = (date: Dayjs) => {
+    setBrowsingMonthStart(date);
+    setSelectedDate(null);
+  };
+
+  const onDaySelect = (date: Dayjs) => {
+    setSelectedDate(date.format("YYYY-MM-DD"));
+  };
+
+  const onTimeSelect = (time: string) => {
+    setSelectedTimeslot(time);
+  };
+
+  useEffect(() => {
+    setLayout(isMobile ? "mobile" : "small_calendar");
+  }, [isMobile, setLayout]);
+
+  useEffect(() => {
+    if (event.isLoading) return setBookerState("loading");
+    if (!selectedDate) return setBookerState("selecting_date");
+    if (!selectedTimeslot) return setBookerState("selecting_time");
+    return setBookerState("booking");
+  }, [event, selectedDate, selectedTimeslot, setBookerState]);
+
+  return (
+    <>
+      {/* @TODO: This is imported because the DatePicker relies on it. Perhaps
+      rewrite the datepicker as well? */}
+      <CustomBranding lightVal={null} darkVal={null} />
```

Comment:
I didn't want to touch or duplicate the datepicker. So for now that means we need to load the custombranding component as well. As soon as we delete the old booker I want to improve the datepicker + this custombranding component as well.